pre-commit/run-org-hooks

```diff
@@ -14,19 +14,21 @@ from pathlib import Path
 orghooks_dir = os.path.dirname(os.path.realpath(__file__))
 
 LINT_CONFIG_FILE = 'tox.ini'
+MODERN_LINT_CONFIG_FILE = 'pyproject.toml'
 
 
 def copy_configs(dest_dir):
-    """ Make a copy of configurations for hooks if one doesn't exist. """
+    """Override configurations for hooks"""
     dev_tools_dir = Path(orghooks_dir).parents[0]
     lint_configs_dir = dev_tools_dir / 'lint-configs'
     shutil.copyfile(lint_configs_dir / LINT_CONFIG_FILE, dest_dir / LINT_CONFIG_FILE)
+    shutil.copyfile(lint_configs_dir / MODERN_LINT_CONFIG_FILE, dest_dir / MODERN_LINT_CONFIG_FILE)
 
 
 def main():
     """ Main entrypoint. """
-    repo_dir = os.getcwd()
-    copy_configs(Path(repo_dir))
+    repo_dir = Path.cwd()
+    copy_configs(repo_dir)
     cfg = os.path.join(orghooks_dir, 'orghooks.yaml')
```

Comment:
can we also have orghooks as a Path just on initialization? All we do with that var are path operations

Reply:
As `os.path.join` seems able to join `Path` objects and strings I guess we can

Reply:
Just got rid of os.path